Rakefile

```diff
@@ -0,0 +1,29 @@
+require 'typhoeus'
+require 'json'
+require 'statsd'
+
+module Services
+  def self.statsd
+    @statsd ||= Statsd.new
+  end
+end
+
+def get(url)
+  response = Typhoeus.get(url)
+  JSON.parse(response.body)
+end
+
+def gauge(stat, value)
+  puts "GAUGE: govuk.tagging.#{stat}: #{value}"
+  Services.statsd.gauge("govuk.tagging.#{stat}", value)
+end
+
+task :run_monitoring do
+  total_items = get("https://www.gov.uk/api/search.json?count=0")
+  gauge "total_items", total_items.fetch("total")
+
+  untagged_items = get("https://www.gov.uk/api/search.json?count=0&filter_taxons=_MISSING")
```

Comment:
Out of curiosity, do we expect literally everything in search to be tagged evenutally, or will there always be some pages which don't have taxons?